packages/rum-recorder/src/boot/startRecording.spec.ts

```diff
@@ -86,6 +86,7 @@ describe('startRecording', () => {
         segment: jasmine.any(File),
         'session.id': 'session-id',
         start: jasmine.stringMatching(/^\d{13}$/),
+        raw_segment_size: jasmine.stringMatching(/^\d+$/),
```

Comment:
> add field raw_segment_size, it will be rename raw_segment_size_in_byte in the intake

what is the reasoning for renaming the field at the intake? could we name it correctly from the SDK?

Reply:
At the beginning it was not but @BenoitZugmeyer did not like having the unit in the variable name and I believe it's useful when used across systems so I rename it in the intake. That way in dogweb I know what I an dealing with

Reply:
I am just curious, do we often rename fields on the intake or is this a rare exception?
In my opinion, it's better to have the same names, but if we are already doing this for many other fields then I think it's ok for this one too.

Reply:
Following discussion it's not renamed anymore. We keep: `raw_segment_size` across systems

Reply:
> I am just curious, do we often rename fields on the intake or is this a rare exception?

No, that was me trying to be explicit.
The convention is to have the sizes in bytes so I will follow convention, no need to add the unit.